dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -1,8 +1,8 @@
 .. _using-sitl-for-ardupilot-testing:
 
-=====================
-SITL Advanced Testing
-=====================
+==========
+Using SITL
```

Comment:
Don't we now have two pages with the same title?

Is one of these pages supposed to essentially be a "quick-start" where the other is a "how to use it to properly test ArduPilot"?

Reply:
yeah as I was changing names I missed this....should be setting up, starting, using....will change...and Invoke is too expensive a word..